### PR TITLE
Reorganize Meeting and Election folders as features

### PR DESCRIPTION
**Pull request by Xelowak**
I reorganized directories for Meeting and Election (E-voting) as I did with the Social Media. I added some documentation and improve clarity of some files along the way.
I also removed some code smells.

### CODE REVIEW

File: fe1-web/features/evoting/network/messages/SetupElection.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@ import {
 import { OpenedLaoStore } from 'store';
 import { ProtocolError } from 'model/network/ProtocolError';
 import { validateDataObject } from 'model/network/validation';
-import { Question } from 'model/objects/Election';
-import { ActionType, MessageData, ObjectType } from '../MessageData';
-import { checkTimestampStaleness } from '../Checker';
+import { Question } from 'features/evoting/objects/Election';
```
Comment (pierluca) [inline fe1-web/features/evoting/network/messages/SetupElection.ts:7]:
```suggestion
import { Question } from '../../objects';
```

File: fe1-web/features/evoting/network/ElectionMessageApi.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import {
+  channelFromIds, EventTags, Hash, Lao, Timestamp,
+} from 'model/objects';
+import { Question, Vote } from 'features/evoting/objects/Election';
```
Comment (pierluca) [inline fe1-web/features/evoting/network/ElectionMessageApi.ts:4]:
```suggestion
import { Question, Vote } from '../objects';
```

File: fe1-web/features/evoting/objects/__tests__/Election.test.ts [hunk 3]
```diff
@@ -1,13 +1,15 @@
 import 'jest-extended';
+
 import '__tests__/utils/matchers';
 import STRINGS from 'res/strings';
 import { mockLaoIdHash, mockLaoName } from '__tests__/utils/TestUtils';
-import { LaoEventType } from '../LaoEvent';
+import { LaoEventType } from 'model/objects/LaoEvent';
+import { Timestamp } from 'model/objects/Timestamp';
+import { Hash } from 'model/objects/Hash';
```
Comment (pierluca) [inline fe1-web/features/evoting/objects/__tests__/Election.test.ts:8]:
```suggestion
import { LaoEventType, Timestamp, Hash } from 'model/objects';
```

**Review by pierluca — COMMENTED** [review]
Minor remarks, but this looks pretty good already !

File: fe1-web/features/evoting/network/ElectionMessageApi.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+ */
+
+/**
+ * Adapts the starting time if start < creation.
+ *
+ * @param start - The start time of the event
+ * @param creation - The creation time of the event
+ */
+const adaptStartTime = (creation: Timestamp, start: Timestamp) => ((start.before(creation))
+  ? creation : start);
```
[the remark this answers is not in the document]
Comment (Xelowak) [reply fe1-web/features/evoting/network/ElectionMessageApi.ts:21]: That's a great idea, I'm on it.

**Review by pierluca — COMMENTED** [review]
Nice cleanup of `Timestamp` !

**Review by pierluca — APPROVED** [review]

**Pull request merged**